### ISSUE TITLE
changing from getElementById to a querySelector

### ISSUE THREAD

**Issue by leumasxxx** [publiclab/Leaflet.DistortableImage#1174]
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from underrepresented groups in free and open-source software!

We know that the process of creating a pull request is one of the biggest barriers for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Claim the issue by commenting. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

<details>
  <summary>💡 <i>Learn how to claim</i> 🙋</summary>
  <h3>Claiming an issue</h3>
  <p>Unless the issue is marked as reserved for someone, you can just say "I'd like to try this!" and then you've claimed it - no need to wait for someone to assign it to you. Just be sure you link your pull request (PR) to this issue so we can see where your solution is.</p>
  <p>And open one early if possible - even before you've completed it with additional commits - and others can help you figure out any issues you may face.</p>
</details>

- [ ] 📝 **Update** the file [pixelorigin.html](https://github.com/publiclab/Leaflet.DistortableImage/blob/f51666389d5d78bf1aea9cd527f7266cb436ca64/debug/pixelorigin.html) in the `Leaflet.DistortableImage` repository (press the little pen Icon) and edit the line as shown below.

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
-90  document.getElementById('info').innerHTML =
+90 document.querySelector('#info').innerHTML =
$DIFF
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

(This issue was created by [First-Timers-Bot](https://github.com/hoodiehq/first-timers-bot).)

**Comment by leumasxxx** [comment]
@TildaDares @cesswairimu I created this FTO for first timers who are still looking for an issue to resolve and I'd be delighted if it can be assigned to someone

**Comment by TildaDares** [comment]
Hi @leumasxxx, thank you for opening this issue for first-timers. Please take a look at this issue to know the expected format of an FTO https://github.com/publiclab/Leaflet.DistortableImage/issues/1135. Thank you!

**Comment by leumasxxx** [comment]
> 

@TildaDares ive seen the format and I've made the necessary adjustments

**Comment by 7malikk** [comment]
@leumasxxx Nice work! Keep them coming 👍🏾

**Comment by liliyao2022** [comment]
Hi, I'd like to try.

**Comment by liliyao2022** [comment]
hi @leumasxxx, the file link is wrong. Please add the correct link. Thanks!

**Comment by leumasxxx** [comment]
> hi @leumasxxx, the file link is wrong. Please add the correct link. @liliyao2022 I do not understand what you mean by incorrect link can you rephrase please?

**Comment by leumasxxx** [comment]
> @leumasxxx Nice work! Keep them coming 👍🏾

@7malikk thank you very much

**Comment by TildaDares** [comment]
Hi @leumasxxx, please add the correct file link in the issue description. Thank you!

**Comment by TildaDares** [comment]
Hi @liliyao2022, an issue was reserved for you here https://github.com/publiclab/Leaflet.DistortableImage/issues/1146

**Comment by leumasxxx** [comment]
> Hi @leumasxxx, please add the correct file link in the issue description. Thank you!
Okay im on it

**Comment by leumasxxx** [comment]
@TildaDares  after going through the issue I created, I noticed my faults in it but now I think I've made the necessary corrections and it's  ready to be assigned.

**Comment by TildaDares** [comment]
Amazing work @leumasxxx. Thank you!

**Comment by TildaDares** [comment]
Reserved for @iLynette for 24 hours!

**Comment by iLynette** [comment]
On it @TildaDares Thank you.

**Comment by TildaDares** [comment]
Hi @iLynette, do you need help with this? Please reach out if you do. Thanks!

**Comment by jssol** [comment]
Hello @iLynette @TildaDares, I would love to help out on this issue.

**Comment by TildaDares** [comment]
Reserved for @Jolah1 for 24 hours!

**Comment by Jolah1** [comment]
@TildaDares working on it now,
thank you.

**Comment by Aarav238** [comment]
Hey, I would like to work on this.

**Comment by Aarav238** [comment]
I'd like to try this!

**Comment by Nehemiah60** [comment]
Hello, I will like to try this issue

**Comment by 7malikk** [comment]
@TildaDares @cesswairimu this looks like it's not being worked on could it be assigned to @Nehemiah60 or anyone in line?

**Comment by Nehemiah60** [comment]
Yes please

**Comment by TildaDares** [comment]
Hi @Nehemiah60, please go ahead. Thanks @7malikk!

**Comment by Nehemiah60** [comment]
Hello, would you like to review? I made a commit. Thanks.

**Comment by 7malikk** [comment]
@Nehemiah60 Make a PR and ping me or any of the mentors, we'd be glad to review and merge.
Thank you!

**Comment by Nehemiah60** [comment]
Okay, based on the instructions provided, I could not edit the file once I clicked it because I was not on the branch. I had to open it in github.dev. made the changes and pushed them to the patch-1 branch. I am stuck in making a PR.

**Comment by 7malikk** [comment]
Hey @Nehemiah60 I believe I just saw your PR #1298 
Guess you already figured it out.

**Comment by Nehemiah60** [comment]
Yeah, I did

**Comment by AliJafriETH** [comment]
Hey @7malikk, i would love to work on this issue, is it still open?

**Comment by 7malikk** [comment]
Hello @AliJafriETH, Sorry this issue is still open, it has already been worked on.
You could be on the lookout for new good first-timer-issues.
Cheers!

**Comment by AliJafriETH** [comment]
Sure @7malikk , thank you man!

**Comment by 7malikk** [comment]
> Sure @7malikk , thank you man!

Anytime @AliJafriETH

**Comment by Sreejith2** [comment]
i would like to fix this issue.

**Comment by pankajkrbansal** [comment]
I'd like to try this!

**Comment by shubham-singh-748** [comment]
just want to know, is this issue still open, if yes then please assign me & i am ready to work on that.....

**Comment by hound77** [comment]
@TildaDares I want to work on this issue

**Comment by hound77** [comment]
@TildaDares I want to contribute can you please help me to get a good first issue ticket to start with.
